clusters/app.ci/prow/03_deployment/retester.yaml

```
@@ -38,6 +58,9 @@ spec:
         - --job-config-path=/etc/job-config
         - --cache-file=/cache/backoff
         - --config-file=/etc/retester/_config.yaml
+        ports:
+          - name: metrics
+            containerPort: 9090
```

To keep the indents' consistency:
```suggestion
        - name: metrics
          containerPort: 9090
```